Dockerfile

```diff
@@ -4,8 +4,8 @@ ARG FFMPEG_VERSION=latest
 FROM node:alpine as web-builder
 ARG JELLYFIN_WEB_VERSION=10.5.1
 RUN apk add curl git \
- && curl -L https://github.com/jellyfin/jellyfin-web/archive/${JELLYFIN_WEB_VERSION}.tar.gz | tar zxf - \
- && cd jellyfin-web-* \
+ && git clone --branch release-10.5.z --single-branch https://github.com/jellyfin/jellyfin-web.git \
```

Comment:
Might be worthwhile to add the `--depth 1` option here to avoid pulling more git history than necessary

Reply:
Could that cause issues? And not sure if it matters much, this is only going to be for point releases